src/FavoriteOutputDefinition/Dao.php

```diff
@@ -69,7 +69,9 @@ public function create()
     public function save()
     {
         if ($this->model->getId()) {
-            return $this->update();
+            $this->update();
```

Comment:
This change is wrong: we need to add `return` here on update and remove all the `void` support from phpdoc instead
https://github.com/pimcore/web2print-tools/blob/8e383d2a72a1fa0c88a7f5cbfb0226c942099a70/src/FavoriteOutputDefinition/Dao.php#L97
Otherwise it's never `bool` because `save()` and `create()` are calling each other until `update` returns a `bool`